### PR TITLE
feat: Study datatypes

### PR DESCRIPTION
**Pull request by Dithinkrishna**
feat: Updated JS-data types and if loops codes

### CODE REVIEW

File: 2-datatypes/boolean.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["number1", "number2"], main)
+function main(err, res) {
+    const number1 = true(res.number1);
```
Comment (dinumathai) [inline 2-datatypes/boolean.js]: Please only one input `prompt.get(["input"], main)`.... once u get that you can compare the text as 
```
let parsedInput = false;
if(res.input === "true"){
  parsedInput = true;
} else if (res.input === "false") {
  parsedInput = false;
} else {
  console.log("invalid input");
  return;
}
```

**Comment by Dithinkrishna** [comment]
Resolved reviewd  codes

File: 2-datatypes/boolean.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["input"], main)
+function main(err, res) {
+    let parsedInput = false;
+    if (res.input === "true") {
+        parsedInput === true;
```
Comment (dinu-mathai) [inline 2-datatypes/boolean.js]: This is not assignment

Comment (Dithinkrishna) [reply 2-datatypes/boolean.js]: sorry
sir I am blocked 😊

can you give me some more hints.

File: 2-datatypes/boolean.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["input"], main)
+function main(err, res) {
+    let parsedinput = false;
+    if (res.input === "true") {
+        console.log(parsedinput = true);
```
Comment (dinu-mathai) [inline 2-datatypes/boolean.js]: Do not do assignment and logging at same line. Do `parsedinput = true` here.

`console.log(parsedinput` at the end of functions.

File: 2-datatypes/boolean.js (hunk 1 above)
Comment (Dithinkrishna) [reply 2-datatypes/boolean.js]: sir ,I am blocked.
Can u ping once you are free.

File: 2-datatypes/boolean.js (hunk 3 above)
Comment (Dithinkrishna) [reply 2-datatypes/boolean.js]: updated codes

**Review by dinu-mathai — APPROVED** [review]

**Review by dinu-mathai — APPROVED** [review]

**Pull request merged**